api/src/main/java/org/openmrs/Concept.java

```diff
@@ -448,9 +448,8 @@ public ConceptName getName(Locale locale) {
 	 */
 	public ConceptName getName() {
 		if (getNames().isEmpty()) {
-			if (log.isDebugEnabled()) {
-				log.debug("there are no names defined for: " + conceptId);
-			}
+				log.debug("there are no names defined for: {}", conceptId);
+			
```

Comment:
please remove the indentation now that the conditional is replaced :) and also please do not add extra newlines.

please apply that comment to all your changes, thank you :)